src/main.rs

```diff
@@ -347,7 +347,12 @@ fn run_sudo_command(
         );
     }
 
-    let mut args = vec!["-SHiu".to_string(), ctx.target_user.clone()];
+    let mut args = vec!["-Hiu".to_string(), ctx.target_user.clone()];
```

Comment:
The `-S` argument is incompatible with the `-A (--askpass)` switch and does not seem necessary. I don't know why I added it to begin with. Hopefully it does not cause regressions.

Per man page:
> -S, --stdin
> Write the prompt to the standard error and read the password from the standard input instead of using the terminal device.